packages/mrmime/mrmime.0.1.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+name:         "mrmime"
+version:      "dev"
```

Comment:
```suggestion
```
This is going to force the version to `dev` and not `0.1.0`. Also the field is unnecessary.